web/settings.py

```diff
@@ -153,7 +162,7 @@ class Common(Configuration):
 
 class Dev(Common):
     DEBUG = True
-    ALLOWED_HOSTS = ["localhost", "127.0.0.1", "[::1]", "0.0.0.0"]
+    ALLOWED_HOSTS = ["localhost", "127.0.0.1", "[::1]", "0.0.0.0", "load-balance-mentoria-1815320676.us-east-1.elb.amazonaws.com"]
```

Comment:
Talvez conseguimos resolver isso utilizando a URL do site. 

```suggestion
    ALLOWED_HOSTS = ["localhost", "127.0.0.1", "[::1]", "0.0.0.0", "load-balance-mentoria-1815320676.us-east-1.elb.amazonaws.com"]
```

Reply:
Eu acabei não vendo esse comentário. Eu acho que dá para só definir a variável `ALLOWED_HOSTS` que a versão em prod [carrega automaticamennte](https://github.com/DadosAbertosDeFeira/maria-quiteria/blob/9c99062c8a69309784af7236d1bdec36e42484c9/web/settings.py#L172).

O problema que você estava tendo é rodando local?